test/test.js

```diff
@@ -35,45 +32,82 @@ describe('VCOnstructor Test', function() {
 });
 
 describe('verifies the set and get operations', function() {
+
```

Comment:
Should we keep these line breaks after every line?
I like keeping all the lines in test without spaces.
there should be line break only for each `describe` and `it`
Whats your opinion on this?

Reply:
Ya sure . i actually  dont have an idea regarding that. i just gave line so that i will be easy.